### PR TITLE
PE-1364: Take the contract readers from core

### PR DESCRIPTION
**Pull request by matibat**

### CODE REVIEW

File: daily_output.base.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		"weekly": {
+			"hasReachedMinimumGroupEffort": false,
+			"groupEffortRewards": [],
+			"streakRewards": []
+		},
+		"lastWeek": {
+			"hasReachedMinimumGroupEffort": false,
+			"groupEffortRewards": [],
+			"streakRewards": []
+		}
```
Comment (karlprieb) [inline daily_output.base.json]: its missing `total`, right?

Comment (karlprieb) [inline daily_output.base.json]: its missing `total`, right?

Comment (matibat) [reply daily_output.base.json]: That was the wrong merge. See #8 (WIP)

Comment (matibat) [reply daily_output.base.json]: That was the wrong merge. See #8 (WIP)

**Review by matibat — COMMENTED** [review]
Self-review checkpoint

**Review by karlprieb — APPROVED** [review]

**Pull request merged**